frontend/packages/ceph-storage-plugin/src/components/modals/advanced-kms-modal/advanced-kms-modal.tsx

```diff
@@ -53,13 +53,29 @@ export const AdvancedKMSModal = withHandlePromise((props: AdvancedKMSModalProps)
   const [error, setError] = React.useState('');
 
   const vaultNamespaceTooltip = t(
-    'ceph-storage-plugin~Vault enterprise namespaces are isolated environments that functionally exist as Vaults within a Vault. They have separate login paths and support creating and managing data isolated to their namespace.',
+    'ceph-storage-plugin~The Vault namespace to use for the OpenShift Container Storage communication with the Vault server. Using relative paths is allowed. Mainly used for KMS tenants.',
   );
 
   const KMSFileSizeErrorMsg = t(
     'ceph-storage-plugin~Maximum file size exceeded. File limit is 4MB.',
   );
 
+  const vaultCACertTooltip = t(
+    `ceph-storage-plugin~A PEM-encoded CA certificate file used to verify the Vault server's SSL certificate. This value, if exists, takes precedence over VAULT_CAPATH.`,
```

Comment:
Do we support setting `VAULT_CAPATH`? That would mean the PATH is inside the Ceph-CSI provisioner and node-plugin, and we normally do not add additional files to our containers. Maybe drop _This value, if exists, takes precedence over VAULT_CAPATH._?